dist/src/main/java/io/camunda/application/initializers/HealthConfigurationInitializer.java

```diff
@@ -86,10 +87,13 @@ protected boolean shouldReadinessState(final List<String> activeProfiles) {
    * Returns a list of health indicators which will be member of the readiness group
    *
    * @param activeProfiles
+   * @param env
    * @return
```

Comment:
🙃 Gotta love this very useful javadoc here 😂 If there's no description it might as well not be here imo.